docs/reference/release-notes/highlights-7.7.0.asciidoc

```diff
@@ -18,3 +18,110 @@ pivot and summarize your data and store it in a new index. See
 {ref}//transform-apis.html[{transform-cap} APIs].
 
 // end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+=== Finer memory control for bucket aggregations
+
+A new setting has been introduced - `search.check_buckets_step_size` - to
+better control how the coordinating node will allocate memory when aggregating
+buckets. The allocation of buckets will now be done in steps, each step
+allocating a number of buckets equal to this setting. A parent circuit breaker
+check is performed on allocation, to avoid an OOM failure.
```

Comment:
```suggestion
allocating a number of buckets equal to this setting. To avoid an `OutOfMemory`
error, a parent circuit breaker check is performed on allocation.
```